core/trino-main/src/main/java/io/trino/operator/scalar/JsonExtract.java

```diff
@@ -118,7 +118,7 @@
 {
     private static final int ESTIMATED_JSON_OUTPUT_SIZE = 512;
 
-    private static final JsonFactory JSON_FACTORY = new JsonFactoryBuilder()
+    private static final JsonFactory JSON_FACTORY = DefaultJsonFactory.builder()
             .disable(CANONICALIZE_FIELD_NAMES)
             .build();
```

Comment:
maybe we can have a method or single json factory field that is shared between all json functions (all 4 java files)?

Or maybe we should define another field in `DefaultJsonFactory` that we can simply use here?

Reply:
I'd prefer to make this as a follow up. Canonicalization could and should be disabled in most of the cases when we are working with the external input.

Reply:
Nevermind. I did introduce a constant for that as well.